content/build/hydrobuild.md

```diff
@@ -451,3 +451,24 @@ default `docker` builder, run the following command:
 ```console
 $ docker context use default
 ```
+
+### How do I manage the build cache with Hydrobuild?
+
+You don't need to manage the builder's cache manually. The system manages it
+for you through [garbage collection](./cache/garbage-collection.md).
+
+Hydrobuild uses the following garbage collection limits:
+
+- 90% of 1TB of data
+- 180 days of history
+- 10 000 builds
```

Comment:
I think both concern the history right? One is a time limit while the other is on the number of entries.